scripts/util/get_y_class.py

```diff
@@ -0,0 +1,83 @@
+"""
+(C) Gregory Way 2016
+PanCancer NF1 Classifier
+scripts/util/get_y_class.py
+
+Description:
+Function to determine nearest neighbor of unlabeled data
+
+Usage:
+Import only by build_y.py
+"""
+
+import os
+import pandas as pd
+
+
+def get_y_class(mut, tis, gene=['NF1'], mut_filt=['Silent'],
```

Comment:
@cgreene I think we ran into this problem at Princeton. If I remember it took me several days to debug. Lesson -- avoid at all costs!